xbmc/pvr/windows/GUIEPGGridContainer.cpp

```diff
@@ -1567,55 +1567,49 @@ void CGUIEPGGridContainer::LoadLayout(TiXmlElement *layout)
   TiXmlElement *itemElement = layout->FirstChildElement("channellayout");
   while (itemElement)
   {
-    CGUIListItemLayout itemLayout;
-    itemLayout.LoadLayout(itemElement, GetParentID(), false, m_width, m_height);
-    m_channelLayouts.push_back(itemLayout);
+    m_channelLayouts.emplace_back();
+    m_channelLayouts.back().LoadLayout(itemElement, GetParentID(), false, m_width, m_height);
     itemElement = itemElement->NextSiblingElement("channellayout");
   }
   itemElement = layout->FirstChildElement("focusedchannellayout");
   while (itemElement)
   {
-    CGUIListItemLayout itemLayout;
-    itemLayout.LoadLayout(itemElement, GetParentID(), true, m_width, m_height);
-    m_focusedChannelLayouts.push_back(itemLayout);
+    m_focusedChannelLayouts.emplace_back();
```

Comment:
Also here - parameter missing? c/p?

Reply:
Do you mean the `emplace_back` parameter? No, this invokes the default constructor, just like the old code did on the temporary (which is eliminated by this patch).